src/main/scala/viper/gobra/ast/internal/Program.scala

```diff
@@ -109,6 +111,7 @@ case class Method(
                  override val results: Vector[Parameter.Out],
                  override val pres: Vector[Assertion],
                  override val posts: Vector[Assertion],
+                 override val terminationMeasure: Option[Vector[Assertion]],
```

Comment:
Here, we could use a `Vector[Assertion]` instead of `Option[Vector[Assertion]]`. The case `None` can be trivially represented by the empty vector.